### PR TITLE
Add notes on installation of libcxx

### PR DESCRIPTION
**Pull request by JoostvanPinxten**
PR for https://github.com/klee/klee/issues/1075, adding a few notes on dependencies and usage.

### CODE REVIEW

File: build-llvm60.md [hunk 1]
```diff
@@ -57,19 +57,27 @@ The current procedure for building KLEE with LLVM 6.0 (recommended) is outlined
...
+   $ sudo pip install wllvm
+   ```
+
    Run from the main KLEE source directory:
 
    ```bash
    $ LLVM_VERSION=6.0 SANITIZER_BUILD= BASE=<LIBCXX_INSTALL_DIR> ./scripts/build/build.sh libcxx
    ```
-   where `<LIBCXX_INSTALL_DIR>` is the directory where you want to install libcxx.
+   where `<LIBCXX_INSTALL_DIR>` is an _absolute_ directory where you want to install libcxx.
```
Comment (251) [inline build-llvm60.md]: I think "absolute path" makes more sense (+ below).

File: build-llvm60.md [hunk 2]
```diff
@@ -57,19 +57,27 @@ The current procedure for building KLEE with LLVM 6.0 (recommended) is outlined
...
 
    To tell KLEE to use libcxx, pass the following flags to CMake when you configure KLEE in step 9:
 
    ```bash
    -DENABLE_KLEE_LIBCXX=ON -DKLEE_LIBCXX_DIR=<LIBCXX_INSTALL_DIR>/libc++-install-60/ -DKLEE_LIBCXX_INCLUDE_DIR=<LIBCXX_INSTALL_DIR>/libc++-install-60/include/c++/v1/
    ```
 
+   Note that `<LIBCXX_INSTALL_DIR>` must currently be an _absolute_ directory, i.e., no relative paths or `~` is allowed.
```
Comment (MartinNowack) [inline build-llvm60.md]: `~` often depends on the current file resolution strategy of your environment.
A workaround is to us `$HOME` instead.

**Comment by JoostvanPinxten** [comment]
Please let me know whether there are any more points of improvement now.

**Comment by ccadar** [comment]
@JoostvanPinxten thanks!  @251 @MartinNowack I assume you're happy to merge this?  I'll do this shortly unless I hear otherwise.

**Pull request merged**